ios/ledgerlivemobile.xcodeproj/project.pbxproj

```diff
@@ -598,13 +598,6 @@
 			remoteGlobalIDString = 274692C321B4414400BF91A8;
 			remoteInfo = "RNSentry-tvOS";
 		};
-		F8A31E45223A3BDF00F81049 /* PBXContainerItemProxy */ = {
-			isa = PBXContainerItemProxy;
-			containerPortal = 64C6A31B337D471B93E32DCD /* RNSentry.xcodeproj */;
-			proxyType = 2;
-			remoteGlobalIDString = 274692C321B4414400BF91A8;
-			remoteInfo = "RNSentry-tvOS";
-		};
```

Comment:
was this accidental?

Reply:
I used add -p but wasn't sure since I reinstalled packages. for now I guess I'll reset it to `develop`'s state